students/rfelts/lesson09/mailroom_oo/donor_models.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+
+# Russell Felts
+# Assignment 9 - Donor Classes
+
+
+import operator
+import datetime
+
+
+class Donor(object):
+    """ Handles all the information about a single donor: attributes, properties, and methods to provide access to the
+    donor-specific information that is needed.  """
+
+    def __init__(self, name, donations=None):
+        """ Initialize the Donor class """
+
+        self.name = name
+        assert type(donations) is list, "\nThe donation value must be entered as a list.\n"
```

Comment:
`assert` isn't typically used in code - instead use `raise Exception`

`assert` is used during debugging or in unit tests.